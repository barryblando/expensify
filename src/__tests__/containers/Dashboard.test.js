import React from 'react';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import { shallow } from 'enzyme';
import MainDashboardPage from '../../containers/DashboardPage';

const mockStore = configureMockStore();
const store = mockStore({});

test('should render Main Dashboard correctly', () => {
  const wrapper = shallow(
    <Provider store={store}>
      <MainDashboardPage expenses={{}} income={{}} />
    </Provider>
  );
  expect(wrapper).toMatchSnapshot();
});
