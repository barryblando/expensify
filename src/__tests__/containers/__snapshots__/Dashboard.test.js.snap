// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Main Dashboard correctly 1`] = `
<Connect(DashboardPage)
  expenses={Object {}}
  income={Object {}}
/>
`;
